core/src/main/java/discord4j/core/object/entity/Message.java

```diff
@@ -990,7 +990,10 @@ public enum Flag {
         SUPPRESS_NOTIFICATIONS(12),
 
         /** This message is a voice message. */
-        IS_VOICE_MESSAGE(13);
+        IS_VOICE_MESSAGE(13),
+
+        /** This message use components v2 */
```

Comment:
TODO: Need mention here and maybe in MessageCreateSpec about how this flag make content/embeds not working for new message